jina/hubble/hubio.py

```diff
@@ -674,9 +678,10 @@ def deploy_public_sandbox(args: Union[argparse.Namespace, Dict]) -> str:
                     headers=get_request_header(),
                 ).json()
 
-                host = json_response.get('data', {}).get('host', None)
-                port = json_response.get('data', {}).get('port', None)
-                livetime = json_response.get('data', {}).get('livetime', '15 mins')
+                data = json_response.get('data') or {}
```

Comment:
```suggestion
                data = json_response.get('data', {})
```

Reply:
this is intended change.
In case of error, `json_response` is `{'data': None, ...}`, then `data` variable becomes `None` instead of `{}`.

Reply:
okk